test/parallel/test-net-listen-anyaddr.js

```diff
@@ -0,0 +1,10 @@
+var assert = require('assert');
+var cp = require('child_process');
+
+var args = [
+  '-e',
+  "require('net').createServer().listen(0, function() { this.close(); });",
+];
+
+// should exit cleanly almost immediately
+cp.execFile(process.execPath, args, {timeout: 500}, assert.ifError);
```

Comment:
It was more or less arbitrary. I know some CI machines can be slow at things like forking, but I wasn't sure _how_ slow. On the upside, the test is not CPU intensive and will run in parallel, so its impact on the suite overall should be _extremely_ low.